docs/troubleshooting.md

```diff
@@ -19,3 +19,18 @@ You recently downloaded or updated XCode.
 2. Choose `Legacy Build System`
 3. Remove `node_modules/react-native/third-party` and
    `node_modules/react-native/third-party-podspecs`
+
+## I get Android crashes when using the Reader SDK
+
+### The problem
+
+When there are different versions of Google Play Services, crashes can occur like
```

Comment:
The problem description is frustratingly nondescript. Can we be more specific about when the crash occurs and _why_ having different versions of the Play Service causes a conflict? Also, "like" implies there are other possible errors that could occur? If that's true, we should provide a list of known errors that occur with this issue. If that's not true, rephrase to make the problem description definitive. For example: "When there are different versions of Google Play Services on the same device, <problem description>, and Android will crash with an <error name> error."